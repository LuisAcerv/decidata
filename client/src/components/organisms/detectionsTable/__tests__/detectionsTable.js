import React from "react";
import { shallow } from "enzyme";
import DetectionsTable from "../";

describe("DetectionsTable", () => {
  it("should render correctly", () => {
    let wrapper;
    wrapper = shallow(<DetectionsTable />);
    expect(wrapper).toMatchSnapshot();
  });
});
