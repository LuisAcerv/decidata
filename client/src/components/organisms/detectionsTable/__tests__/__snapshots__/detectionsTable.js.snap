// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DetectionsTable should render correctly 1`] = `ShallowWrapper {}`;
